result/yoda.js

```diff
@@ -0,0 +1,19 @@
+
+/*eslint no-multi-spaces: "error"*/
+
+var a =  1;
+var a =  1;
+var a =  1;
+
+if(foo   === "bar") {}
```

Comment:
__[development][eslint]__
* Error - 'foo' is not defined. (no-undef)
* Error - Multiple spaces found before '==='. (no-multi-spaces)
* Error - Empty block statement. (no-empty)
<!-- Comment by [SideCI](https://sideci.com) -->